bundle/metadata/annotations.yaml

```diff
@@ -4,7 +4,7 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: aws-load-balancer-operator
-  operators.operatorframework.io.bundle.channels.v1: stable-v1.2,stable-v1
+  operators.operatorframework.io.bundle.channels.v1: stable-v1.3,stable-v1
```

Comment:
Another thing to be checked with OLM.  Do we still need to add chanbels here? This is redundant with FBC channels.